### PR TITLE
Nightmode

### PR DESCRIPTION
**Pull request by BKBetz**
finished nightmode and added new icon

### CODE REVIEW

File: scss/partials/_vars.scss [hunk 1]
```diff
@@ -16,4 +16,5 @@
     --red-fee: #e4354e;
     --circle-red: #f9f3ef;
     --overdue-red: #e64960;
+    --black: #000000;
 }
```
Comment (Vijay421) [inline scss/partials/_vars.scss]: Add enter.

File: js/darkmode.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                document.documentElement.style.setProperty('--red', '#e22944');
+                document.documentElement.style.setProperty('--red-fee', '#e4354e');
+                document.documentElement.style.setProperty('--circle-red', '#f9f3ef');
+                document.documentElement.style.setProperty('--overdue-red', '#e64960');
+                document.documentElement.style.setProperty('--black', '#000000');
+            }
+
+
+        });
+}
```
Comment (Vijay421) [inline js/darkmode.js]: Add enter.

File: js/darkmode.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+window.night = true;
+function switchToDark() {
+
+
+    document.querySelector('.night')
+        .addEventListener('click', function () {
```
Comment (Vijay421) [inline js/darkmode.js]: replace function to arrow function.

**Review by JasperSpillenaar — COMMENTED** [review]
Change the vh in navbar_section.

**Review by Vijay421 — APPROVED** [review]

**Review by StanBrabander — APPROVED** [review]

**Pull request merged**